docs/node/mainnet/README.md

```diff
@@ -27,6 +30,7 @@ Feel free to use other seed nodes besides the one provided here.
   * [23.0.9](https://github.com/oasisprotocol/oasis-core/releases/tag/v23.0.9)
 * [Oasis Rosetta Gateway](https://github.com/oasisprotocol/oasis-rosetta-gateway) version:
   * [2.6.0](https://github.com/oasisprotocol/oasis-rosetta-gateway/releases/tag/v2.6.0)
+  * Public Rosetta endpoint: `https://rosetta.oasis.dev`
```

Comment:
I believe this does not belong here. The primary audience of the network parameters page are node operators that need to know which versions to run. They are potential candidates providing such endpoints, not users of public endpoints. We should probably have docs on how to set up a Rosetta Gateway, similar to Web3 Gateway.

Reply:
Oops, this is a leftover, sorry. Existing deployments are on the dApp page, because dApp devs will be using them.